gulpfile.babel.js

```diff
@@ -135,6 +135,7 @@ function watch() {
   gulp.watch(PATHS.assets, copy);
   gulp.watch('src/pages/**/*.html').on('all', gulp.series(pages, browser.reload));
   gulp.watch('src/{layouts,partials}/**/*.html').on('all', gulp.series(resetPages, pages, browser.reload));
+  gulp.watch('src/data/**/*').on('all', gulp.series(resetPages, pages, browser.reload));
```

Comment:
Panini does only process `.js, .json, .yml` files as i can see: https://github.com/zurb/panini/blob/master/lib/loadData.js#L11

So we could only watch these file extensions. 

Could look like: 
```js
gulp.watch('src/data/**/*.{js,json,yml}').on('all', gulp.series(resetPages, pages, browser.reload));
```

Reply:
I @marvinhuebner 👋. Sorry for the late reply. You are right, I'll close this PR in favor of yours.